test/e2e/dapr_version/sample/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "dapr-e2e-test",
+    "version": "1.0.0",
+    "description": "e2e test to show successful build of dapr",
+    "main": "dist/index.js",
+    "private": true,
+    "scripts": {
+        "build": "rimraf ./dist && tsc",
+        "start": "npm run build && node dist/index.js",
+        "start:dapr-grpc": "dapr run --app-id e2e-test --app-protocol grpc npm run start",
+        "start:dapr-http": "dapr run --app-id e2e-test --app-protocol http npm run start"
+    },
+    "keywords": [],
+    "license": "ISC",
+    "devDependencies": {
+        "rimraf": "^3.0.2",
+        "ts-node": "^9.1.1",
+        "typescript": "^4.2.4"
+    },
+    "dependencies": {
+        "@dapr/dapr": "2.4.1",
```

Comment:
This should come from the local repository (similar to our examples), so that problems can be caught in the build step itself. You can try breaking a type (similar to the issues we've had earlier), and `npm run build` should fail.